sdk/src/Services/S3/Custom/Model/GetObjectResponse.cs

```diff
@@ -56,14 +56,25 @@ public partial class GetObjectResponse : StreamResponse
 
         private string bucketName;
         private string key;
-                
+        private long bytesTransferred = 0;
+
         /// <summary>
         /// Flag which returns true if the Expires property has been unmarshalled
         /// from the raw value or set by user code.
         /// </summary>
         private bool isExpiresUnmarshalled;
         internal string RawExpires { get; set; }
 
+        /// <summary>
+        /// Gets and sets the TransferredAlreaddy property. 
```

Comment:
property name is `BytesTransferred` but the docs says it's `TransferredAlreaddy`

Reply:
Fixed it